content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -190,40 +190,40 @@ Rechenschaftspflicht ist wichtig, deshalb wollen wir dieses Jahr mit der Veröff
 * Überprüfen Sie, wer sich auf Ihrem Server anmeldet und in welchen Räumen Ihr Server Mitglied ist.  
-* Überprüfen Sie die Berichte Ihrer Nutzer und ergreifen Sie Maßnahmen, um gemeldete schädliche Inhalte zu entfernen. Überprüfen Sie Ihre rechtlichen Verpflichtungen in dem Land, in dem Sie Ihren Server betreiben.  
+* Überprüfen Sie die Meldungen Ihrer Nutzer und ergreifen Sie Maßnahmen, um gemeldete schädliche Inhalte zu entfernen. Überprüfen Sie Ihre rechtlichen Verpflichtungen in dem Land, in dem Sie Ihren Server betreiben.  
 * Arbeiten Sie mit anderen Serverbetreibern zusammen, um Informationen über schädliche Räume und Nutzer auszutauschen. Sie können sich an unser Trust & Safety Team unter [abuse@matrix.org](mailto:abuse@matrix.org) wenden, um dieses Gespräch zu beginnen.
 
 ### Zivilgesellschaftliche Gruppen, Wissenschaft und Industrieverbände
 
-Wir begrüßen die Unterstützung von Gruppen der Zivilgesellschaft und Akademikern, die in diesem Bereich arbeiten. Bitte wenden Sie sich an [abuse@matrix.org](mailto:abuse@matrix.org) und teilen Sie uns Ihre Kontaktdaten und Ihr Interessengebiet mit, damit wir uns austauschen können. Wir sind sehr daran interessiert, vertrauenswürdige Berichterstatter hinzuzuziehen. Wenn Sie uns also Berichte schicken möchten, lassen Sie es uns bitte wissen. Um die Ausbreitung von Online-Missbrauch zu bekämpfen, müssen alle Teile des Puzzles zusammenarbeiten, und wir wollen ein gutes Beispiel dafür sein, wie Technik für die Gesellschaft arbeiten kann. Wir würden gerne mit Branchengruppen zusammenarbeiten, Erfahrungen austauschen und voneinander lernen. Die Eintrittsbarrieren für diese Gruppen sind eine Herausforderung, und wir würden uns freuen, wenn Sie uns helfen würden, sie abzubauen.
+Wir begrüßen die Unterstützung aus zivilgesellschaftlichen Gruppen und der Wissenschaft, die in diesem Bereich arbeiten. Bitte wenden Sie sich an [abuse@matrix.org](mailto:abuse@matrix.org) und teilen Sie uns Ihre Kontaktdaten und Ihr Interessengebiet mit, damit wir uns austauschen können. Wir sind sehr daran interessiert, vertrauenswürdige Berichterstatter hinzuzuziehen; wenn Sie uns also Berichte schicken möchten, melden Sie sich bitte bei uns. Um die Ausbreitung von Online-Missbrauch zu bekämpfen, müssen alle Teile des Puzzles zusammenarbeiten, und wir wollen ein gutes Beispiel dafür sein, wie Technik eine positive Auswirkung auf die Gesellschaft haben kann. Wir würden gerne mit Branchengruppen zusammenarbeiten, Erfahrungen austauschen und voneinander lernen. Die Eintrittsbarrieren für diese Gruppen sind eine Herausforderung, und wir würden uns freuen, wenn Sie uns helfen würden, sie abzubauen.
```

Comment:
- "zivi"/"Wissenschaft": match section title
- "melden Sie sich bei uns": "let us know" may work in English but is much too colloquial for the otherwise rather formal tone ("Sie") of the German version, so here is a slighly more fitting variant
- "positive Auswirkung": the former literal translation sounded extremely Germanised at best, so here is a more native suggestion